src/test/java/org/verdictdb/coordinator/RedshiftTpchSelectQueryCoordinatorTest.java

```diff
@@ -1132,7 +1123,7 @@ public void TpchQuery21Test() {
           }
         }
       }
-      */
+      TpchQuery21Test();
```

Comment:
Why is this calling itself again?

Reply:
Oh, that should be a mistake.

Reply:
Anyway, I find query 21 has count distinct in having and count in the select list. I think in this situation, this should not be supported because we will convert having into select list.